accumulo-store/src/main/java/gaffer/accumulostore/key/core/AbstractCoreKeyAccumuloElementConverter.java

```diff
@@ -272,8 +270,13 @@ public Properties getPropertiesFromColumnVisibility(final String group, final by
             if (null != propertyDef) {
                 final Serialisation serialiser = propertyDef.getSerialiser();
                 try {
-                    properties.put(schema.getVisibilityProperty(),
-                            serialiser.deserialise(columnVisibility));
+                    final Object columnVisibilityProperty;
+                    if (columnVisibility == null || columnVisibility.length == 0) {
+                        columnVisibilityProperty = "";
```

Comment:
The visibility property could be any Object - it may not be a string, so unfortunately we can't default it to an empty string.